blog-posts/host-listener/host-listener.md

```diff
@@ -0,0 +1,76 @@
+---
+published: false
+title: "Ain't nobody needs HostListener"
+cover_image: ""
+description: "In Angular event handling is often implemented using the hostListener decorator, even though it might not be the best fit for the problem. Have you consider more composable approaches?"
+tags: angular, rxjs, javascript, typescript
+series:
+canonical_url:
+---
+
+Angular's `@hostListener` is well known within the community. Rather unknown are the problems this might have on runtime performance and general application architecture. In general, there are two main problems with the `hostListener` decorator.
```

Comment:
"two main problems" > there are three problems listed, or did you want to say "a few"?

Reply:
there used to be two at the beginning :D